roles/common/clone_repo/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Clone the repository
+  git:
+    repo: "{{ github_repo }}"
```

Comment:
Suggest changing the variable name to better clarify that it's a cloneable `.git` address (and that the role could take a repo from bitbucket or something else that isn't github).

`git_clone_url` or `git_clone_address` or something along those lines maybe?

Reply:
`git_clone_address`